### PR TITLE
docs(halyard_jars): Add docs for how to install halyard via jars.

### PR DESCRIPTION
**Pull request by brandonnelson3**

### CODE REVIEW

File: setup/install/halyard_migration.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
```
Comment (dorbin) [inline setup/install/halyard_migration.md:1]: I think you'll need to either add a sentence in halyard.md that refers to this doc or add this to _data/navigation.yml. Or  both? Unless you only want this page accessible by direct link, like if someone asks us a question about migrating and we send them the link.

Comment (brandonnelson3) [reply setup/install/halyard_migration.md:1]: Lars was of the mindset of people finding out about this page only when they actually needed to migrate. So I added a check in the install script that basically says, if you already have halyard installed from the debian package, give the user the link to this page.

You can see that change here: https://github.com/spinnaker/halyard/pull/806

Comment (dorbin) [reply setup/install/halyard_migration.md:1]: Ah yes, I remember now. Perf. Approved.

**Review by dorbin — APPROVED** [review]

File: setup/install/halyard_migration.md (hunk 1 above)
Comment (dorbin) [reply setup/install/halyard_migration.md:1]: "Perf" here meaning "perfect." ;)

Comment (brandonnelson3) [reply setup/install/halyard_migration.md:1]: ;) We can always add that link you are suggesting later if it actually be comes a thing we get asked about though. Thanks for the review! This ended up much better IMO than when I first sent it.

**Pull request merged**